cloud/passportauth/integration/index.ts

```diff
@@ -0,0 +1,33 @@
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
+import * as express from 'express';
+import {PassportAuth, PassportSetup, UserApiService, UserSecService} from '../src/index';
+import {UserDataSource} from './UserDataSource';
+import userSeedData, {User} from './UserSeedData';
+
+const userSource = new UserDataSource<User>(userSeedData);
+const userApi = new UserApiService<User>(userSource);
+const userSec = new UserSecService<User>(userApi);
+const passportSetup = new PassportSetup<User>(userSec);
+const authService = new PassportAuth<User>(userSec);
+const app = express();
+
+app.use(bodyParser.json());
+app.use(cors());
+passportSetup.init(app); // initialize passport
+
+app.get('/testSecureEndpoint', authService.protect('test', '/login'), (req: express.Request, res: express.Response) => {
```

Comment:
For 1.0, we will just use `authService.protect()` with just one param - a string of the role to protect the route with. 
eg. `authService.protect('testRole')`